packages/zoe/test/unitTests/contracts/test-secondPriceAuction.js

```diff
@@ -293,27 +304,45 @@ test('zoe - secondPriceAuction w/ 3 bids - alice exits onDemand', async t => {
     // Bob bids
     const bobSeat = await zoe.offer(bobInvitation, bobProposal, bobPayments);
 
-    t.rejects(
-      () => E(bobSeat).getOfferResult(),
-      /The item up for auction is not available or the auction has completed/,
-      'The bid should have failed.',
+    // Bob exits early, but this does not stop the auction.
+    bobSeat.tryExit();
```

Comment:
Did it work? Bob should check to see if he got results or payouts. Since he didn't specify `onDemand`, I didn't expect this to work. Does tryExit() return anything on failure?

Reply:
The default exit rule is `onDemand` so that is what Bob has. He is able to exit with refunds, and we do check his payouts

Reply:
We do check later. I'd like the test to show that he got his deposit back immediately, and what his offer result looks like.

Reply:
Ok, sure, the offer result will be the usual one. I will move the deposits up

Reply:
Done